rpcs3/Emu/RSX/GL/GLGSRender.cpp

```diff
@@ -335,9 +335,7 @@ void GLGSRender::on_init_thread()
 
 	if (!m_overlay_manager)
 	{
-		m_frame->hide();
 		m_shaders_cache->load(nullptr);
-		m_frame->show();
```

Comment:
These should be unchanged. There's nothing gained but risking regressions

Reply:
Fixed.